README.md

```diff
@@ -29,24 +31,28 @@ Execute installation script
     1. Sunbird RC
     2. Esignet
     0. Exit
-    Select: 
+    Select:
     ```
 
 3. Select "Sunbird RC" as the first step of the installation process.
 
 4. The installation will encompass the following services:
-
    * [Credential Schema](https://github.com/Sunbird-RC/sunbird-rc-core/tree/main/services/credential-schema)
    * [Credential Service](https://github.com/Sunbird-RC/sunbird-rc-core/tree/main/services/credentials-service)
    * [Identity Service](https://github.com/Sunbird-RC/sunbird-rc-core/tree/main/services/identity-service)
    * [Registry](https://github.com/Sunbird-RC/sunbird-rc-core)
 5. Post Sunbird installation, proceed to create an issuer and credential schema. Refer to the API schemas available [here](https://github.com/Sunbird-RC/sunbird-rc-core/tree/main/api-documentation).
-6. Add the jar file of Digital Credential Stack(DCS) plugin implementation in [loader_path](docker-compose-esignet/loader_path) the GitHub link for the repository can be found [here](https://github.com/mosip/digital-credential-plugins).
+    * Set the hostname of the endpoints correctly as per your docker setup
+    * Now generate a DID, create a credential schema and create an issuance registry
+        * take note of $.schema[0].author & $.schema[0].id from the create credential schema request
+6. Add the jar file of Digital Credential Stack(DCS) plugin implementation in [loader_path](docker-compose-esignet/loader_path) the GitHub link for the repository can be found [here](https://github.com/mosip/digital-credential-plugins) if you wish to build it locally or you can download from [here](https://mvnrepository.com/artifact/io.mosip.esignet.sunbirdrc/sunbird-rc-esignet-integration-impl).
```

Comment:
links are interchanged

Reply:
can we add the steps for vc download as well? @vharsh

Reply:
I've reworked the above sentence to make it less confusing.